upload.py

```diff
@@ -145,8 +145,6 @@ def check_missing_data(asset_type, props):
                 "   Please check the filepath and try again.",
             )
 
-    if asset_type == "HDR":
```

Comment:
Odstraneni IF zpusobi, ze Else na radku 159 se nikdy nevykona, protoze pro cokoliv, co neni HDR nebo BRUSH se udela jen if asset_type not in ("HDR", "BRUSH") a dalsi else if uz se nevyhodnocuji. Je potreba ponechat if asset_type == "HDR":pass.

```
else:
        if props.thumbnail == "":
            write_to_report(
                props,
                "A thumbnail image has not been provided.\n"
                "   Please add a thumbnail in JPG or PNG format, ensuring at least 1024x1024 pixels.",
)
```
<img width="276" alt="Screenshot 2024-02-06 at 11 53 18" src="https://github.com/BlenderKit/blenderkit/assets/22000268/4c63fd2c-2fb1-429c-846e-b6876f6296ef">

Reply:
reverted that change. in commit 8190752